src/components/molecules/NavSearchBar/NavSearchBar.tsx

```diff
@@ -141,6 +143,20 @@ const NavSearchBar = () => {
     [searchQuery]
   );
 
+  const focusSearchBar = useCallback((e) => {
+    e.preventDefault();
+    const searchBar = document.getElementById("NavSearchBar__search-input");
+    if (searchBar) searchBar.focus();
+    // TODO: we need to remove "f" from being entered into the input which would happen now
+  }, []);
+
+  useMousetrap({
+    keys: ShortcutKeys.find,
+    callback: focusSearchBar,
+    // TODO: bindRef: (null as never) as MutableRefObject<HTMLElement>,
+    withGlobalBind: true, // TODO: remove this once we have a ref to bind to
+  });
```

Comment:
Given these are intended to be global key bindings, I wonder if we should be setting them up somewhere more globally?

Not sure how we would achieve that in an easy/efficient way though at this stage..

@mike-lvov Any thoughts? I know I pushed back on you including key bindings in a 'deeper' component in the past (though that was related to map keyboard controls that already had a location they were being set/called/etc, so maybe this is a different case)

Reply:
well, I guess I could define ```useMousetrapGlobal``` as an adapter over to useMousetrap setting `withGlobalBind: true`. is that you are hinting on @0xdevalias ?

Reply:
Nah, was more thinking in the same way that the movement controls are all wrapped up in a single hook (useKeyboardControls) currently, can we centralise everything else in a similar way.

I feel like the answer is likely no/doesn’t make sense to; so I expect that centralising the shortcut definitions themselves in src/settings (as described elsewhere on my review comments) is probably the best case scenario